spec/messages.js

```diff
@@ -17,6 +17,7 @@
                 ['rosterGroupsFetched', 'chatBoxesFetched'], {},
                 async function (done, _converse) {
 
+            _converse.whitelisted_plugins = _converse.whitelisted_plugins.filter(e => e !== 'converse-forward-message');
```

Comment:
The second parameter of `initConverse` 3 lines above is a settings map.
So you can pass in `{blacklisted_plugins: ['converse-forward-message']}` there instead of doing this here.

Reply:
I tried it. Then the test fails again. The problem is, that I check with the following if the plugin is enabled:
`if (!_converse.whitelisted_plugins.includes('converse-forward-message'))`
So when I add it to the blacklisted plugins it is also a whitelisted plugin. That's why it will not reject the message and the test fails. I'm thinking about also adding `|| _converse.blacklisted_plugins.includes('converse-forward-message')` to the if-statement. I don't really like the solution anyways, but could not find a better way from preventing converse to send an error-message on forwarded-messages, if the plugin is enabled.

I also tried to find out why the tests for xep-0363. For me all tests work and I cannot find anything in the code (by just looking) that jumps out. I updated the pull-Request anyway to the latest commit.